app/controllers/content_items_controller.rb

```diff
@@ -9,6 +9,8 @@ def show
     else
       render text: 'Not found', status: :not_found
     end
+  rescue GdsApi::HTTPForbidden => e
+    render text: e.message, status: 403
```

Comment:
This might be better handled with a `rescue_from` call at controller level.  See https://github.com/alphagov/collections/blob/master/app/controllers/subtopics_controller.rb#L6 for an example (in that example, `error_404` is a method defined in `ApplicationController`).